### PR TITLE
[Custom DC] Add docs about prepare/upload data

### PR DESCRIPTION
**Pull request by shifucun**
Start to add sections in each page. It's not the complete version.

### CODE REVIEW

File: custom_dc/prepare_data.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+title: Prepare Data
+nav_order: 2
+parent: Custom Data Commons
+published: true
+---
+
+## Overview
+
+Preparing data involves cleaning / formatting the raw data into compatible CSV
+file. The CSV file is expected to have columns corresponding to the Values
```
Comment (beets) [inline custom_dc/prepare_data.md:12]: nit: ... CSV files. Each CSV file is ...

File: custom_dc/prepare_data.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+CSV files (_.csv).
+
+## General CSV format
+
+In the table shown below, there are separate columns for Variable (Variable),
+Place (Country), Date (Year) and Value (Value) and each row of the CSV
+corresponds to one observation of the Variable about a Place at the specified
+Date.
+
+| Year | Country | Variable        | Value       | Extra Column |
```
Comment (beets) [inline custom_dc/prepare_data.md:26]: Add a note about Extra Column? Or use brackets to separate it as optional

File: custom_dc/prepare_data.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+| ---- | ------- | --------------- | ----------- | ------------ |
+| 2017 | UK      | Life_Expectancy | 81.25609756 | 1            |
+| 2017 | UK      | Population      | 65844142    | 2            |
+
+The TMCF for this CSV looks like:
+
+```txt
+Node: E:CSV_FILE_NAME->E0
```
Comment (beets) [inline custom_dc/prepare_data.md:34]: Also add a note about the format here? CSV_FILE_NAME might be used verbatim from the example.

**Review by beets — COMMENTED** [review]
small nits...

Also, perhaps @lucy-kind can help review for clarity

File: custom_dc/upload_data.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    │   │   │   └── foo.csv
+    │   │   ├── schema.mcf
+    │   │   └── provenance.json
+    │   └── source2/
+    │       ├── file_group1/
+    │       │   ├── baz.tmcf
+    │       │   └── baz.csv
+    │       ├── schema.mcf
+    │       └── provenance.json
+    ├── cache
```
Comment (Fructokinase) [inline custom_dc/upload_data.md:39]: As discussed offline, do cache, control, go under "internal" under "gcs_folder"?

File: custom_dc/upload_data.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    ├── control
+    └── provenance.json
+```
+
+Raw data should be uploaded under `/<project>/data/<import>/<dataset>`. Each
```
Comment (Fructokinase) [inline custom_dc/upload_data.md:44]: update the path here to match the above?

File: custom_dc/upload_data.md (hunk 4 above)
Comment (shifucun) [reply custom_dc/upload_data.md]: updated

File: custom_dc/upload_data.md (hunk 5 above)
Comment (shifucun) [reply custom_dc/upload_data.md]: done

File: custom_dc/prepare_data.md (hunk 3 above)
Comment (shifucun) [reply custom_dc/prepare_data.md]: changed to data as it does not matter or need to match file name

**Review by Fructokinase — APPROVED** [review]

**Comment by shifucun** [comment]
thanks for review!

**Pull request merged**